spec/models/remote_theme_spec.rb

```diff
@@ -468,6 +468,136 @@ def about_json(
+                    love: "FFFFFF",
+                    tertiary_low: "000000",
+                  },
+                  scheme2: {
+                    love: "AACCDD",
+                    tertiary_low: "99AAFF",
+                  },
+                },
+              )
+              .to_json,
+        )
+
+        theme = RemoteTheme.import_theme(initial_repo_url)
+        scheme1 = theme.color_schemes.find_by(name: "scheme1")
+        scheme2 = theme.color_schemes.find_by(name: "scheme1")
+        expect(scheme1.base_scheme_id).to eq(nil)
+        expect(scheme2.base_scheme_id).to eq(nil)
+
+        expect do
+          ColorSchemeRevisor.revise(
+            scheme1,
+            { colors: [{ name: "love", hex: "111111" }] },
+            diverge_from_remote: true,
+          )
+        end.to change { ColorScheme.unscoped.count }.by(1)
+
+        scheme1_base_scheme_id = scheme1.reload.base_scheme_id
+        expect(scheme1_base_scheme_id).to be_present
+        expect(scheme1.colors.find_by(name: "love").hex).to eq("111111")
+        expect(
+          ColorScheme.unscoped.find(scheme1_base_scheme_id).colors.find_by(name: "love").hex,
+        ).to eq("ffffff")
+        expect(scheme2.base_scheme_id).to eq(nil)
+
+        add_to_git_repo(
+          initial_repo,
+          "about.json" =>
+            JSON
+              .parse(about_json)
+              .merge(
+                color_schemes: {
+                  scheme1: {
+                    love: "EEEEEE",
+                    tertiary_low: "000000",
+                  },
+                  scheme2: {
+                    love: "AACCDD",
+                    tertiary_low: "99AAFF",
+                  },
+                },
+              )
+              .to_json,
+        )
+        theme.remote_theme.update_from_remote
+
+        scheme1.reload
+        expect(scheme1.base_scheme_id).to eq(scheme1_base_scheme_id)
+        expect(scheme1.colors.find_by(name: "love").hex).to eq("111111")
+        expect(
+          ColorScheme.unscoped.find(scheme1_base_scheme_id).colors.find_by(name: "love").hex,
+        ).to eq("eeeeee")
+        expect(scheme2.base_scheme_id).to eq(nil)
+      end
+
+      it "deletes color schemes that haven't been modified and deletes the bases of schemes that have been modified" do
+        add_to_git_repo(
+          initial_repo,
+          "about.json" =>
+            JSON
+              .parse(about_json)
+              .merge(
+                color_schemes: {
+                  scheme1: {
+                    love: "FFFFFF",
+                    tertiary_low: "000000",
+                  },
+                  scheme2: {
+                    love: "AACCDD",
+                    tertiary_low: "99AAFF",
+                  },
+                },
+              )
+              .to_json,
+        )
+
+        theme = RemoteTheme.import_theme(initial_repo_url)
+
+        expect(theme.color_schemes.pluck(:name)).to contain_exactly("scheme1", "scheme2")
```

Comment:
```suggestion
```

This line is unnecessary given that we are already fetching both `scheme1` and `scheme2` records.